tox.ini

```diff
@@ -1,9 +1,38 @@
 [tox]
-envlist = py34,py35,py36,py37
+envlist =
+    clean
+    django22-py{37,36}
+    django21-py{37,36,35}
+    django20-py{36,35,34}
+    report
+
 [flake8]
 max-line-length = 120
 exclude = tests/fixtures/*
+
 [testenv]
-deps = pytest
-commands =
+deps =
     pytest
+    pytest-cov
+    django20: {[django]2.0.x}
+    django21: {[django]2.1.x}
+    django22: {[django]2.2.x}
+
+commands =
+    py.test --cov=./flake8_django --cov-append
+
+[testenv:clean]
+deps = coverage
+skip_install = true
+commands = coverage erase
+
+[testenv:report]
+skip_install = true
+deps = coverage
+commands =
+    coverage report --fail-under=100
```

Comment:
This report part is not needed, since coverage reports are being uploaded to codecov.